draft-ietf-quic-multipath.md

```diff
@@ -902,68 +879,23 @@ Path ID (e.g. more than half of the available Path ID are used).
 
 ## Number Spaces
 
-As stated in {{introduction}}, when multipath is negotiated, each
+As stated in {{introduction}}, when the multipath extension is negotiated, each
 path uses a separate packet number space.
 This is a major difference from
 {{QUIC-TRANSPORT}}, which only defines three number spaces (Initial,
 Handshake and Application packets).
 
-Implementation of multipath capable QUIC will need to carefully
-model the relations between paths and number spaces, as shown
-in {{fig-number-spaces}}.
-
-~~~
-   +-------------------------+
-   | CID received from peer: |-- - - - - - +
-   | Sender Number Space     |             |
-   +-------------------------+             v
-                      ^             +----------------+
-                      |             | Path (4 tuple) |
-                      +-------------| - RTT          |
-                +------------------>| - Congestion   |
-                |                   |   state        |
-                v                   +----------------+
-   +-------------------------+             ^
-   | CID provided to peer:   |             |
-   | Receiver Number Space   |-- - - - - - +
-   +-------------------------+
-~~~
-{: #fig-number-spaces title="Send and Receive number spaces"}
-
-The path is defined by the 4-tuple through which packets are
-received and sent. Packets sent on the path will include the
-Destination Connection ID currently used for that path, selected
-from the list of connection IDs provided by the peer. Packets received
-on the path carry a Destination CID selected by the peer from
-the list provided to that peer.
```

Comment:
This was removed as number spaces are now per path ID and not changes with the CID.